charts/argo-cd/values.yaml

```
@@ -225,6 +225,9 @@ configs:
     #     -----BEGIN CERTIFICATE-----
     #     ... encoded certificate data here ...
     #     -----END CERTIFICATE-----
+    #   allowedAudiences:
+    #     - aaaabbbbccccddddeee
+    #     - qqqqwwwweeeerrrrttt
```

nits; It's better to add comments to align upstream's doc? 
Ref: https://argo-cd.readthedocs.io/en/stable/operator-manual/user-management/#existing-oidc-provider
```suggestion
    # Optional list of allowed aud claims. If omitted or empty, defaults to the clientID value above (and the 
    # cliClientID, if that is also specified). If you specify a list and want the clientID to be allowed, you must 
    # explicitly include it in the list.
    # Token verification will pass if any of the token's audiences matches any of the audiences in this list.
    #   allowedAudiences:
    #     - aaaabbbbccccddddeee
    #     - qqqqwwwweeeerrrrttt
```

---

Would it also make sense to add the comments relating to other `oidc.config` keys to align with upstream?